### PR TITLE
pyenv: add example to list all installable versions

### PR DESCRIPTION
**Pull request by mfrw**
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

PS: This is my first attempt at a translation page. I used `Google Translate`
Help wanted for `zh` page though :)

### CODE REVIEW

**Review by marchersimon — APPROVED** [review]
Looks good, thanks

**Comment by marchersimon** [comment]
> PS: This is my first attempt at a translation page. I used `Google Translate`

Oh, wait. Do you speak Italian? Because we'd rather have an outdated translation than a machine translated one.

**Comment by mfrw** [comment]
> > PS: This is my first attempt at a translation page. I used `Google Translate`
> 
> Oh, wait. Do you speak Italian? Because we'd rather have an outdated translation than a machine translated one.

Do I speak Italian ? ... Nope :(
It was my first attempt at a translation page. Please be brutal here. I have no idea what i've submited.

**Comment by marchersimon** [comment]
> I have no idea what i've submited.

Well, that's usually not a good starting point :smile:

Either the translation is translated or verified by a human, or we cannot take it.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Coudld we request a review from someone who does speak Italian

**Comment by marchersimon** [comment]
The only one I can think of is @mebeim

**Comment by mfrw** [comment]
I reverted that change :)
So maybe no review required

**Comment by mebeim** [comment]
FWIW, correct Italian translation would be: 

> Elenca tutte le versioni di Python che possono essere installate da upstream:

One could also leave out the "da upstream" since it's not really translatable to Italian.

File: pages/common/pyenv.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 `pyenv versions`
 
+- List all Python versions which can be installed from upstream:
```
Comment (marchersimon) [inline pages/common/pyenv.md:14]:
```suggestion
- List all Python versions that can be installed from upstream:
```

**Pull request merged**